### ISSUE TITLE
An empty username is allowed

### ISSUE THREAD

**Issue by jbainbridge98** [QihanLu/CS419#9]
Not a super big deal, but when you select the Add function, you can just hit enter and add a user with no real username. 
![Screenshot (15)](https://user-images.githubusercontent.com/56710599/144776548-b469fa00-5e54-45d6-8e1d-932d48fe7a68.png)

**Comment by Kagesss** [comment]
Yes sure. However as what you said, there is no issues will be caused by this situation so I don't think it is really a bug, but leave it to the grader.

**Comment by symxmyz233** [comment]
I think the name of a user can't be an empty string. But It won't influence the function. **So, it's not a bug but i will give point on the bug finder.**
